seed/challenges/01-front-end-development-certification/object-oriented-and-functional-programming.json

```diff
@@ -188,7 +188,7 @@
         "  //Change this constructor",
         "  this.wheels = 4;",
         "  this.seats = 1;",
-        "  this.engines = 1;",
+        "  this.engines = 5;",
```

Comment:
This should be changed back to `this.engines = 1` and the line above it should be changed to `this.seats = 5`.